lms/djangoapps/courseware/masquerade.py

```diff
@@ -69,6 +69,30 @@ def __setstate__(self, state):
         """
         self.__init__(**state)
 
+    @staticmethod
+    def get_active_group_name(data):
+        """
+        Lookup the active group name
+        """
+        name = None
+        for group in data['available']:
```

Comment:
There are more efficient ways to do this, besides traversing the collection [1].
But with the short-circuit logic, the small collection size (usually <10), and the simple comparison of primitve types (`string`s and `int`s), I'm not too worried about this (`simplicity > complexity`) :shrug: 

- [1] We _could_ check the active group as we add each item to the list (prior to this processing), but then we'd need to re-write that to explicitly support this; seems wonky...